app/scripts/metamask-controller.js

```diff
@@ -7402,7 +7402,7 @@ export default class MetamaskController extends EventEmitter {
           'SnapController:get',
         ),
         trackError: (error) => {
-          return captureException(error);
+          return global.sentry?.captureException?.(error);
```

Comment:
`captureException` doesn't seem to work in E2E tests. Changing it for `global.sentry?.captureException` still works in production, and also works in the tests.